pkg/plugins/common/kustomize/v2/scaffolds/internal/templates/config/samples/crd_sample.go

```diff
@@ -55,11 +55,8 @@ const crdSampleTemplate = `apiVersion: {{ .Resource.QualifiedGroup }}/{{ .Resour
 kind: {{ .Resource.Kind }}
 metadata:
   labels:
-    app.kubernetes.io/name: {{ lower .Resource.Kind }}
-    app.kubernetes.io/instance: {{ lower .Resource.Kind }}-sample
-    app.kubernetes.io/part-of: {{ .ProjectName }}
-    app.kubernetes.io/managed-by: kustomize
-    app.kubernetes.io/created-by: {{ .ProjectName }}
+    app.kubernetes.io/name: {{ .ProjectName }}
+	app.kubernetes.io/managed-by: kustomize
```

Comment:
@sarthaksarthak9 see that it has an space here when should not
Please, fix it, re-run, make generate and then squash all commits so that we can move forward with this one.